playbooks/roles/setup-caasp-workers/handlers/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+# ansible-lint: we need to reload the profiel and there is no ansible modules for that yet
+- name: Reload apparmor profile
+  shell: "apparmor_parser -r /etc/apparmor.d/{{ item.item }}"
+  when: item.changed
+  loop: "{{ apparmor_copy.results | flatten(levels=1) }}"
+  loop_control:
+    label: "{{ item.item }}"
```

Comment:
nit: Not sure why you would want to set the label to this, it doesn't improve readability.

Reply:
I misread I thought this was loop_var.